charts/spire/charts/spire-server/templates/tornjak-service.yaml

```diff
@@ -15,8 +15,12 @@ spec:
   selector:
    {{- include "spire-server.selectorLabels" . | nindent 4 }}
   ports:
-    - name: {{ include "spire-tornjak.backend" . }}
-      port: {{ .Values.tornjak.service.port }}
-      targetPort: tornjak
+    - name: tornjak-srv-http
+      port: {{ .Values.tornjak.service.portHttp }}
+      targetPort: tornjak-http
+      protocol: TCP
```

Comment:
If https is enabled, do we want http exposed out the service if the user specifies type=LoadBalancer? Will it just redirect to https, or will it allow bypassing mtls restrictions?

Reply:
The idea was to make a permanent redirect from HTTP to HTTPS port, so the HTTP service will be always open but redirecting to HTTPS

Reply:
https://github.com/spiffe/tornjak/pull/282/files#diff-7c41928cb703314c59d418bbec523718010421f052461c7e02e88fd3b689433cR612

This is how it is mitigated. This is also a UX feature, so if the consumer forgets to type https in front of the url, this permanent redirect will do it on behalf of the client.